secrets/azurekeyvault/akv.go

```diff
@@ -20,7 +20,10 @@
 //
 // For secrets.OpenKeeper, azurekeyvault registers for the scheme "azurekeyvault".
 // The default URL opener will use Dial, which gets default credentials from the
-// environment.
+// environment, unless the AZURE_KEYVAULT_AUTH_VIA_CLI environment variable is
```

Comment:
nit: `AZURE_KEYVAULT_AUTH_VIA_CLI` is defined by us, so it might be helpful to differentiate it from azure's env names. Maybe starts with `GOCDK`.

Reply:
We don't do that with any of the other GOCDK environment variables, so I don't think we should do it here. There was some discussion earlier about namespacing them that way, but we decided not to.